src/initialize.cc

```diff
@@ -93,6 +95,24 @@ LibraryInitializer::LibraryInitializer()
     cpu_worker_nthreads_(dmlc::GetEnv("MXNET_CPU_WORKER_NTHREADS", 1)),
     mp_cv_num_threads_(dmlc::GetEnv("MXNET_MP_OPENCV_NUM_THREADS", 0)) {
   dmlc::InitLogging("mxnet");
+
+#if !(defined(_WIN32) || defined(_WIN64) || defined(__WINDOWS__))
```

Comment:
maybe it should be moved to a function like install_pthread_atfork_handlers - it probably will be more descriptive on what's happening here